tls/README.md

```diff
@@ -0,0 +1,14 @@
+This content was generated from a live cluster.
+It may change over time and is not guaranteed to be stable.
+This is a useful starting point for understanding the cert chains in openshift used to secure kubernetes.
```

Comment:
Super important is that we need to run this against a "release" cluster which has the cert rotation at 30 days, which usually happens after we cut a branch. This CAN'T be master since that has short rotation.